packages/commonwealth/client/scripts/views/pages/ComponentsShowcase/components/Paginations.showcase.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import CWPagination from 'views/components/component_kit/new_designs/CWPagination';
 
 const PaginationsShowcase = () => {
-  const handleChange = (
+  const handleChange = async (
```

Comment:
action error

Reply:
I think I had fixed this previously, action error meaning it will log an error when used? Or you mean we shouldn't be changing this?